### PR TITLE
feat(translation): caching - pt

### PR DESCRIPTION
**Pull request by dlvideira**
Translation of caching 

- Translate caching pattern to portuguese
- Issue #2285

### CODE REVIEW

File: localization/pt/caching/README.md [hunk 1]
```diff
@@ -0,0 +1,326 @@
+---
+title: Caching
```
Comment (iluwatar) [inline localization/pt/caching/README.md:2]: This should be translated to Portugese as well

File: localization/pt/caching/README.md [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+category: Behavioral
+language: en
```
Comment (iluwatar) [inline localization/pt/caching/README.md:4]: Should be `pt`

**Review by iluwatar — CHANGES_REQUESTED** [review]

File: localization/pt/caching/README.md (hunk 1 above)
Comment (dlvideira) [reply localization/pt/caching/README.md:2]: Changed to Cache.

**Review by iluwatar — APPROVED** [review]

**Comment by iluwatar** [comment]
Looks good! Thank you for the contribution 🎉

@all-contributors please add @dlvideira for translation

**Pull request merged**